tls/client-tls-resume.c

```diff
@@ -275,6 +281,8 @@ int main(int argc, char** argv)
     /* Cleanup and return */
     if (ssl)
         wolfSSL_free(ssl);      /* Free the wolfSSL object              */
+    if (sslRes)
+        wolfSSL_free(sslRes);   /* Free the wolfSSL object              */
 #ifdef OPENSSL_EXTRA   
     if (session)
         wolfSSL_SESSION_free(session);
```

Comment:
This shouldn't be guarded. Using `get1` means that the session always has to be free'd.

Reply:
That's a NULL check. So if `wolfSSL_get1_session` failed, then wouldn't need to free, right?